### DISCUSSION TITLE
Celestia flatpaks from Fedora registry

### DISCUSSION THREAD

**Discussion by mattiaverga** [CelestiaProject/Celestia#1988] [Show and tell]
I'd like to announce that Celestia flatpaks are available from Fedora Linux registry, so that all Linux users can easily install Celestia on their workstation.

To enable Fedora registry just add the repository with the following command:
`sudo flatpak remote-add --if-not-exists fedora-flatpaks oci+https://registry.fedoraproject.org`

Then just search for "Celestia" in your favourite app manager (Discover, Gnome-software...). Fedora currently provides Celestia 1.7 prereleases in GTK and QT6 frontends.

**Comment by 375gnu** [comment]
Thanks!

**Comment by faveoled** [comment]
Full install:
```
sudo flatpak remote-add --if-not-exists fedora-flatpaks oci+https://registry.fedoraproject.org
sudo flatpak install space.celestia.celestia_gtk # or space.celestia.celestia_qt6
```
Removal:
```
sudo flatpak uninstall space.celestia.celestia_gtk # or space.celestia.celestia_qt6
sudo flatpak remote-delete fedora-flatpaks
```

**Comment by jonhbs** [comment]
Hello. Celestia was removed from flatpak :(

**Comment by mattiaverga** [comment]
Yeah, sorry, I'm afraid I had to retire celestia from Fedora because I've recently re-analysed licensing of CelestiaContent and found many resources to have incompatible licenses.
Since it is not possible to remove only the offending files I had to retire everything.

I'll monitor the effort of CelestiaContent to move resources to FOSS licenses and I'll be happy to add back Celestia into Fedora repositories (and flatpak too). Meanwhile, someone else could package celestia in Flathub, which doesn't have licensing restriction which Fedora has.
